coda.io.apexdomain.json

```diff
@@ -0,0 +1,37 @@
+{  
+    "providerId":"coda.io",
+    "providerName":"Coda",
+    "serviceId":"apexdomain",
+    "serviceName":"Coda Site",
+    "version": 1,
+    "syncPubKeyDomain" : "coda.io",
```

Comment:
Looks like some templates use a dedicated subdomain named something like `domainconnect.coda.io` for public keys. Might be a good idea for us to use that pattern too, here and in the other template.

If I understand correctly, the keys themselves would live in TXT records at a subdomain like `<keyId>.domainconnect.coda.io`

Reply:
yes